### ISSUE TITLE
media_player.play_media media_content_type: custom no response

### ISSUE THREAD

**Issue by tunerooster** [custom-components/alexa_media_player#1195]
From: Home Assistant -> Developer Tools -> Service

I enter:

```
Service: media_player.play_media
entity_id: media_player.my_echo_show
media_content_id: 'what time is it?'
media_content_type: custom
```

When I click "CALL SERVICE" nothing happens.  I.e., I am sitting next to the echo show and it is silent.  The "CALL SERVICE" button flashes green, indicating the service is called OK.

With DEBUG logging on, the only relevant message in the log is:
```
2021-02-23 06:20:52 DEBUG (MainThread) [homeassistant.components.websocket_api.http.connection] [140562015356528] Received {'type': 'call_service', 'domain': 'media_player', 'service': 'play_media', 'service_data': {'entity_id': 'media_player.my_echo_show', 'media_content_id': 'what time is it', 'media_content_type': 'custom'}, 'id': 168}
```
I also ran:

```
Service: media_player.play_media
entity_id: media_player.my_echo_show
media_content_id: shuffle greenday
media_content_type: SPOTIFY
```
and it worked just fine (started playing Green Day).  I assume that means I am properly authenticated.

I am running "Alexa Media Player" v3.8.2 on HomeAssistant v2021.2.1

Alexa Media Player was installed using HACS Integrations (using 2FA) and my yaml files have no mention of alexa.

What can I do to troubleshoot this?

Thanks!

P.S.  I plan to run:
```
media_content_id: 'open firefox'
```
which should work, right?  ...although I tried that also and it did nothing.

**Comment by alandtse** [comment]
Since this is on the forum, I'll leave it up there. Otherwise, you'll need to fill out the template.

**Comment by tunerooster** [comment]
Sorry for my ignorance, but can you tell me what you mean by fill out the template.  :)

**Comment by tunerooster** [comment]
Thank you for your reply!  In filling out the template, it asked for my version of alexapy, which I then discovered, I do not have the current cersion 1.24.2.  (I had 1.16 installed).  I used HACS to install Alexa Media Player, and I encountered no mention of alexapy.  Is a HACS installation supposed to upgrade alexapy?

In any case, I removed my old version of alexapy and reinstalled alexa media player, and it now works!

Thanks, your template alerted me to the problem!
